WMF Framework/Remote Notifications/RemoteNotificationsController.swift

```diff
@@ -191,6 +196,26 @@ public enum RemoteNotificationsControllerError: Error {
         operationsController.markAllAsRead(languageLinkController: languageLinkController, completion: completion)
     }
     
+    /// Asks server to mark all notifications as seen for wikipedias that contain local unread notifications.
+    public func markAllAsSeen(completion: @escaping ((Result<Void, Error>) -> Void)) {
+        guard !areFiltersEnabled else {
+            return
```

Comment:
I think it might be good practice to call the completion block here too, just for the sake of consistency. You could add a new case for it to `RemoteNotificationsControllerError` and pass that through like the guards below.

I'm a little torn because it's not _really_ an error, and not really a success either, but I think I lean more towards passing back an error in this case.